db/models.py

```diff
@@ -14,3 +14,36 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre)
```

Comment:
add related names for actors and genres

Reply:
Thanks for your review! I've added all changes to my code.